monitoring/managing-targets.adoc

```diff
@@ -0,0 +1,24 @@
+[id="managing-targets"]
+= Managing targets
+include::modules/common-attributes.adoc[]
+:context: managing-targets
+
+toc::[]
+
+{product-title} Monitoring collects metrics from targeted cluster components by scraping data from exposed service endpoints. In the Administrator perspective in the {product-title} web console, you can use the Targeting UI to view, search, and filter the services and infrastructure components that are currently targeted for scraping. The Targeting UI shows targets for both default {product-title} projects as well as user-defined projects. 
```

Comment:
Should change as the following:
{product-title} Monitoring collects metrics from targeted cluster components by scraping data from exposed endpoints. In the Administrator perspective in the {product-title} web console, you can use the Targeting UI to view, search, and filter the endpoints that are currently targeted for scraping. The Targeting UI shows targets for both default {product-title} projects as well as user-defined projects.

Reply:
Fixed per your recommendation